2025-03-03 14:41:18,450 - INFO - ✅ Using GPU device 1.
2025-03-03 14:41:18,452 - INFO - 📁 Results will be saved in: /caefs/user/mmingyeong/structura/Structura/src/results
2025-03-03 14:41:19,080 - ERROR - Unhandled exception occurred:
Traceback (most recent call last):
  File "converter_ex.py", line 4, in <module>
    import structura
  File "/caefs/user/mmingyeong/structura/Structura/src/structura/__init__.py", line 19, in <module>
    from .SimulationDataConverter import SimulationDataConverter  # ✅ 추가
ModuleNotFoundError: No module named 'structura.SimulationDataConverter'
